plugins/wobbly/wobbly.cpp

```diff
@@ -532,8 +530,7 @@ class wf_wobbly : public wf::view_transformer_t
     }
 
   public:
-    wf_wobbly(wayfire_view view, const wf::plugin_grab_interface_uptr& _iface)
-        : iface(_iface)
+    wf_wobbly(wayfire_view view, const wf::plugin_grab_interface_uptr&)
```

Comment:
since the interface is no longer needed, maybe we can just remove the parameter altogether?

Reply:
lol sorry, when I see a constructor in a plugin type thing, I immediately assume it has to conform to some interface from the plugin system :D